pkg/server/convert.go

```diff
@@ -62,7 +62,7 @@ func TranslationToData(schema translatev1.Schema, translation *model.Translation
 	case translatev1.Schema_JSON_NGX_TRANSLATE:
 		to = convert.ToNgxTranslate
 	case translatev1.Schema_POT:
```

Comment:
```suggestion
	case translatev1.Schema_PO:
```

Reply:
I reverted back, all changes regarding renaming pot -> po. in pkgs, references, files, etc.
git diff was acting silly.

better to do that in separate PR

Reply:
created issue #229